task_29/Anton.Panciuc/src/main/java/com/hillel/homework/Cesar.java

```diff
@@ -0,0 +1,42 @@
+package com.hillel.homework;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cesar {
+  List<Character> alphabet = new ArrayList<>();
+
+  Cesar() {
+    for (char c = 'A'; c <= 'z'; c++) {
+      alphabet.add(c);
+    }
+  }
+
+  String encrypt(String text) {
```

Comment:
Since "encrypt" and "decrypt" methods are utility methods and you actually don't have any fields in the class (alphabet doesn't count, because it is a constant, which you could have defined that way), you can make those methods static and don't need to create instances of a Cesar class afterwards.